Detections/SymantecProxySG/UserAccessedSuspiciousURLCategories.yaml

```diff
@@ -0,0 +1,22 @@
+id: fb0f4a93-d8ad-4b54-9931-85bdb7550f90
+name: User Accessed Suspicious URL Categories
+description: |
+  'Creates an incident in the event the requested URL accessed by the user has been identified as Suspicious, Phishing, or Hacking.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Syslog
```

Comment:
connectorId should be the data concector ID which is SymantecProxySG

Reply:
Corrections made and pushed

Reply:
This still shows requiredDataConnectors:
  - connectorId: Syslog
- datatypes: blank
Please fix this.

Reply:
corrections made and pushed